protostar/starknet/cheatable_syscall_handler.py

```diff
@@ -142,6 +145,16 @@ def _call_contract(
         else:
             raise NotImplementedError(f"Unsupported call type {syscall_name}.")
 
+        # region Modified Starknet code.
+        contract_call_data = (int(str(request.function_selector)), calldata)
```

Comment:
```suggestion
        contract_calldata = (int(str(request.function_selector)), calldata)
```
for naming consistency

Reply:
[typo](https://github.com/software-mansion/protostar/pull/1060/commits/20d9cd1147882005ac9297286cab6b4a2f1826b7)